docs/source/installation.rst

```diff
@@ -2,147 +2,123 @@ Installation
 
-     virtualenv venv-ivadomed --python=python3.6
-
-Step 2: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        ::
 
+            pip install --upgrade pip
 
-::
+            pip install ivadomed
 
-    git clone https://github.com/ivadomed/ivadomed.git
-    cd ivadomed
- 
+    .. tab:: Repo Installation (Advanced or Developer)
 
-Step 3: Install PyTorch 1.5 and TorchVision
-+++++++++++++++++++++++++++++++++++++++++++
+        Bleeding-edge developments are available on the project's master branch
+        on Github. Installation ``ivadomed`` from source:
 
-If you have a compatible NVIDIA GPU that supports CUDA, run the following command:
+        ::
 
-::
-   
-   pip install -r requirements_gpu.txt
+            git clone https://github.com/ivadomed/ivadomed.git
 
+            cd ivadomed
 
-If you do not have a compatible GPU, run the following installer to use ``ivadomed`` with CPU.
+            pip install -e .
 
 
-::
+Step 3: Install ``ivadomed`` with CPU or GPU Support, Install ``torch`` and ``torchvision``
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    pip install -r requirements.txt
+.. tabs::
 
+    .. tab:: CPU Support
 
-Step 4: Install from release (recommended)
-++++++++++++++++++++++++++++++++++++++++++
+        If you plan to run ``ivadomed`` on CPU only, run the following command:
+        ::
 
-Install ``ivadomed`` and its requirements from
-`Pypi <https://pypi.org/project/ivadomed/>`__:
+            pip install -r requirements.txt
 
-::
 
-    pip install --upgrade pip
-    pip install ivadomed
+    .. tab:: Nvidia GPU Support
 
-(Optional) Alternative Step 4 for Developers: Install from source
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`_, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.
 
-Bleeding-edge developments are available on the project's master branch
-on Github. Installation procedure is the following at repository root:
+        If you have a compatible NVIDIA GPU that supports CUDA11 and with the right driver installed, try run the following command:
 
-::
+        ::
 
-    cd ivadomed
-    pip install -e .
+           pip install -r requirements_gpu.txt
 
+        Please note that this must happens after the previous ``ivadomed`` installation step.
 
-(Optional) Step 5 for Developers Install pre-commit hooks
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+Additional Steps Required For ``ivadomed`` Developers: Additional Dependencies and ``pre-commit``
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
This is a nitpicky thing, but what if we had the headers for this section be:

```rest
Developer-only installation steps
+++++++++++++++++++++++++++++++++

The additional steps below are only necessary for contributors to the ``ivadomed`` project.

Additional dependencies and ``pre-commit``
------------------------------------------

Blah blah blah...

``macOS``-specific development steps
------------------------------------

Blah blah blah...
```

That way, the titles can be a little shorter/easier to read, especially in the table-of-contents.

![Screenshot from 2021-11-05 12-25-30](https://user-images.githubusercontent.com/16181459/140544587-5a774cec-b232-4172-a8e4-0997adcf8131.png)

Reply:
Love it! I just couldn't recall the Restructure text heading hierarchy.

Reply:
> Love it! I just couldn't recall the Restructure text heading hierarchy.

The interesting thing is that RST doesn't actually have a heading hierarchy! You can use the symbols in any order, and `docutils`/`sphinx` will interpret each new symbol as the next-smallest header. ([source](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections))

> Normally, there are no heading levels assigned to certain characters as the structure is determined from the succession of headings.

This can get a bit weird, though, because header symbols will often end up inconsistent between pages. :sweat_smile:

Reply:
Thanks for the in-depth explanation.